### ISSUE TITLE
Pixel Block Sorting

### ISSUE THREAD

**Issue by IceandFire04** [HJfod/BetterEdit#168]
<h1>
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/HJfod/BetterEdit/assets/24266948/2cec62bc-d984-4557-981f-759a88606d83">
    <source media="(prefers-color-scheme: light)" srcset="https://github.com/HJfod/BetterEdit/assets/24266948/d9b063f2-24dc-4727-9dc7-c9663f05d9aa">
    <img alt="GitHub Octicons: Beaker" width="100" align="left" src="https://github.com/HJfod/BetterEdit/assets/24266948/d9b063f2-24dc-4727-9dc7-c9663f05d9aa" margin="0px">
  </picture>
  Suggestion
  <br />
</h1>
Suggest a new feature to be added.
<hr />

## Feature Description
Some way to sort pixel blocks, whether it's in the pallete, a new tab, a menu, etc

### How should it work?
I think what would be the most useful is if the block pallete has buttons like the delete menu that opened up submenus where you could select blocks based on similarities

### What does it look/sound/act like?
Basically described above

### What is its purpose?
Make pixel blocks more intuitive and less lobotomizing
